app/host/attendance/Dashboard.tsx

```diff
@@ -124,16 +138,180 @@ export default function Dashboard({
 			</div>
 			<div className="bg-base-100 rounded-xl m-3 ml-1.5 outline outline-base-200 outline-1 basis-1/2 flex flex-col justify-between items-center pl-5 pr-5">
 				<div className="w-full">
-					<div className="flex flex-row w-full justify-between mt-4">
-						<h1 className="text-4xl font-bold">
-							{joined.length}/{totalAttendees} Attendees Present
-						</h1>{" "}
-						<form action={endSession.bind(null, attendanceCode)}>
-							<button className="btn btn-dangerous">End Session</button>{" "}
-						</form>
+					<div className="flex flex-row w-full justify-between mt-4 align-center">
+						<div className="flex flex-col">
+							<h1 className="text-4xl font-bold">
+								{getInClass(joined).length}/{totalAttendees} Attendees Present
+							</h1>{" "}
+							<p className="opacity-50 text-2xl mt-2">
+								{getForeign(joined).map((x) => x.status === 0).length}/
+								{getForeign(joined).length} Foreign Attendees Joined are Present
+							</p>{" "}
+						</div>
+						<div className="flex items-center flex-row">
```

Comment:
This is my first-pass diff-only review. As such, I will not look thoroughly over site styling changes.